brave/src/main/java/brave/internal/propagation/StringPropagationAdapter.java

```diff
@@ -70,7 +70,10 @@
  * }</pre>
  *
  * @since 5.12
+ * @deprecated As of Brave 5.18, throw an {@link UnsupportedOperationException} in
```

Comment:
this is internal, but I put a deprecated message on it anyway as many have been using it without shading. The pull requests I mentioned move off it.